use meson to build

yetist opened the pull request:
Use meson to build:
```
meson _build --prefix /usr
ninja -C _build
```

Note: 
`XXX.xml.in.meson` file is copy of `XXX.xml.in.in`, but change `"<_name></_name>"` to `"<name></name>"`.

raveit65 commented:
Uninstalling locale files doesn't work.
```
[rave@mother mate-backgrounds-dist]$ sudo ninja -C build uninstall
ninja: Entering directory `build'
[0/1] /usr/bin/meson --internal uninstall
Deleted: /usr/local/share/mate-background-properties/mate-abstract.xml
Deleted: /usr/local/share/mate-background-properties/mate-desktop.xml
Deleted: /usr/local/share/mate-background-properties/mate-nature.xml
Deleted: /usr/local/share/backgrounds/mate/abstract/Arc-Colors-Transparent-Wallpaper.png
Deleted: /usr/local/share/backgrounds/mate/abstract/Flow.png
Deleted: /usr/local/share/backgrounds/mate/abstract/Gulp.png
Deleted: /usr/local/share/backgrounds/mate/abstract/Silk.png
Deleted: /usr/local/share/backgrounds/mate/abstract/Spring.png
Deleted: /usr/local/share/backgrounds/mate/abstract/Waves.png
Deleted: /usr/local/share/backgrounds/mate/desktop/Float-into-MATE.png
Deleted: /usr/local/share/backgrounds/mate/desktop/GreenTraditional.jpg
Deleted: /usr/local/share/backgrounds/mate/desktop/MATE-Stripes-Dark.png
Deleted: /usr/local/share/backgrounds/mate/desktop/MATE-Stripes-Light.png
Deleted: /usr/local/share/backgrounds/mate/desktop/Stripes.png
Deleted: /usr/local/share/backgrounds/mate/desktop/Ubuntu-Mate-Cold-no-logo.png
Deleted: /usr/local/share/backgrounds/mate/desktop/Ubuntu-Mate-Dark-no-logo.png
Deleted: /usr/local/share/backgrounds/mate/desktop/Ubuntu-Mate-Radioactive-no-logo.png
Deleted: /usr/local/share/backgrounds/mate/desktop/Ubuntu-Mate-Warm-no-logo.png
Deleted: /usr/local/share/backgrounds/mate/nature/Aqua.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/Blinds.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/Dune.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/FreshFlower.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/Garden.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/GreenMeadow.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/LadyBird.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/RainDrops.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/Storm.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/TwoWings.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/Wood.jpg
Deleted: /usr/local/share/backgrounds/mate/nature/YellowFlower.jpg
Deleted: /usr/local/share/backgrounds/mate/nature
Deleted: /usr/local/share/backgrounds/mate/desktop
Deleted: /usr/local/share/backgrounds/mate/abstract
Deleted: /usr/local/share/backgrounds/mate
Deleted: /usr/local/share/backgrounds
Deleted: /usr/local/share/mate-background-properties
Could not delete /usr/local/share: [Errno 39] Directory not empty: '/usr/local/share'.

Uninstall finished.

Deleted: 36
Failed: 1

Remember that files created by custom scripts have not been removed.
```
eg.
```
[rave@mother mate-backgrounds]$ ls /usr/local/share/locale/zh_CN/LC_MESSAGES/
mate-backgrounds.mo
```
Rest looks good.

yetist commented:
I use the follow commands to test:
```
meson _build
ninja -C _build/
sudo ninja -C _build/ install
sudo ninja -C _build/ uninstall
```
But I didn't get that uninstall error.
```
Deleted: /usr/local/share/backgrounds/mate/desktop                                                                                                                      
Deleted: /usr/local/share/backgrounds/mate/abstract                                                                                                                     
Deleted: /usr/local/share/backgrounds/mate                                                                                                                              
Deleted: /usr/local/share/backgrounds                                                                                                                                   
Deleted: /usr/local/share/mate-background-properties                                
                                                                                    
Uninstall finished.                                                                 
                                                                                    
Deleted: 36                                                                         
Failed: 0                                                                                                                                                               

Remember that files created by custom scripts have not been removed.
```

yetist commented:
Maybe the version is different:
```
$ meson --version
0.50.1
$ ninja --version
1.9.0
```

yetist commented:
Confirmed that the mo file was not uninstalled.
It seems a bug of meson, please see `meson-logs/install-log.txt`, it didn't record the mo files.

yetist commented:
The meson bug found here: https://github.com/mesonbuild/meson/issues/1601

raveit65 commented:
Ok, than i consider this as *known issue* which can't be fixed by our side.
It affect only a minority of users, doesn't exists if package will be installed by a package-manager and autotools installation can be used as workaround.
I think it should be OK to merge it.

hanetzer commented:
> Use meson to build:
> 
> ```
> meson _build --prefix /usr
> ninja -C _build
> ```
> 
> Note:
> `XXX.xml.in.meson` file is copy of `XXX.xml.in.in`, but change `"<_name></_name>"` to `"<name></name>"`.

If you port the main autotools build from intltool (deprecated) to gettext (what you should be using
these days) you can share the file.

yetist commented:
> If you port the main autotools build from intltool (deprecated) to gettext (what you should be using
> these days) you can share the file.

Please see #23

yetist commented:
Rebase on master, and remove .xm.in.meson files.

raveit65 commented:
@yetist 
I re-checked to create a tarball with this PR.
Same problem like in marco PR.
```
[rave@mother mate-backgrounds-dist]$ ls -l mate-backgrounds-1.22.0/po/
insgesamt 884
-rw-rw-r-- 1 rave rave  1297 14. Mai 09:00 af.gmo
-rw-rw-r-- 1 rave rave  3250 14. Mai 09:00 af.po
-rw-rw-r-- 1 rave rave  2006 14. Mai 09:00 am.gmo
-rw-rw-r-- 1 rave rave  3577 14. Mai 09:00 am.po
-rw-rw-r-- 1 rave rave  1191 14. Mai 09:00 an.gmo
-rw-rw-r-- 1 rave rave  3193 14. Mai 09:00 an.po
-rw-rw-r-- 1 rave rave  1504 14. Mai 09:00 ar.gmo
-rw-rw-r-- 1 rave rave  3473 14. Mai 09:00 ar.po
-rw-rw-r-- 1 rave rave  1527 14. Mai 09:00 as.gmo
-rw-rw-r-- 1 rave rave  3490 14. Mai 09:00 as.po
-rw-rw-r-- 1 rave rave  1335 14. Mai 09:00 ast.gmo
-rw-rw-r-- 1 rave rave  3300 14. Mai 09:00 ast.po
-rw-rw-r-- 1 rave rave  1632 14. Mai 09:00 be.gmo
-rw-rw-r-- 1 rave rave  3564 14. Mai 09:00 be.po
-rw-rw-r-- 1 rave rave  2219 14. Mai 09:00 bg.gmo
-rw-rw-r-- 1 rave rave  3792 14. Mai 09:00 bg.po
-rw-rw-r-- 1 rave rave  1524 14. Mai 09:00 bn.gmo
-rw-rw-r-- 1 rave rave  1690 14. Mai 09:00 bn_IN.gmo
-rw-rw-r-- 1 rave rave  3656 14. Mai 09:00 bn_IN.po
-rw-rw-r-- 1 rave rave  3487 14. Mai 09:00 bn.po
-rw-r--r-- 1 rave rave   217 14. Mai 09:00 boldquot.sed
-rw-rw-r-- 1 rave rave  1624 14. Mai 09:00 br.gmo
-rw-rw-r-- 1 rave rave  3513 14. Mai 09:00 br.po
-rw-rw-r-- 1 rave rave  1335 14. Mai 09:00 bs.gmo
-rw-rw-r-- 1 rave rave  3253 14. Mai 09:00 bs.po
-rw-rw-r-- 1 rave rave  1782 14. Mai 09:00 ca.gmo
-rw-rw-r-- 1 rave rave  3376 14. Mai 09:00 ca.po
-rw-rw-r-- 1 rave rave  1801 14. Mai 09:00 ca@valencia.gmo
-rw-rw-r-- 1 rave rave  3381 14. Mai 09:00 ca@valencia.po
-rw-rw-r-- 1 rave rave     0  1. Feb 2018  ChangeLog
-rw-rw-r-- 1 rave rave  1307 14. Mai 09:00 cmn.gmo
-rw-rw-r-- 1 rave rave  3183 14. Mai 09:00 cmn.po
-rw-rw-r-- 1 rave rave   600 14. Mai 09:00 crh.gmo
-rw-rw-r-- 1 rave rave  3085 14. Mai 09:00 crh.po
-rw-rw-r-- 1 rave rave  1891 14. Mai 09:00 cs.gmo
-rw-rw-r-- 1 rave rave  3476 14. Mai 09:00 cs.po
-rw-rw-r-- 1 rave rave  1387 14. Mai 09:00 cy.gmo
-rw-rw-r-- 1 rave rave  3287 14. Mai 09:00 cy.po
-rw-rw-r-- 1 rave rave  1752 14. Mai 09:00 da.gmo
-rw-rw-r-- 1 rave rave  3328 14. Mai 09:00 da.po
-rw-rw-r-- 1 rave rave  1778 14. Mai 09:00 de.gmo
-rw-rw-r-- 1 rave rave  3514 14. Mai 09:00 de.po
-rw-rw-r-- 1 rave rave   668 14. Mai 09:00 dz.gmo
-rw-rw-r-- 1 rave rave  3129 14. Mai 09:00 dz.po
-rw-rw-r-- 1 rave rave  2037 14. Mai 09:00 el.gmo
-rw-rw-r-- 1 rave rave  3645 14. Mai 09:00 el.po
-rw-rw-r-- 1 rave rave  1350 14. Mai 09:00 en_AU.gmo
-rw-rw-r-- 1 rave rave  3236 14. Mai 09:00 en_AU.po
-rw-r--r-- 1 rave rave  1337 14. Mai 09:00 en@boldquot.header
-rw-rw-r-- 1 rave rave   636 14. Mai 09:00 en_CA.gmo
-rw-rw-r-- 1 rave rave  3100 14. Mai 09:00 en_CA.po
-rw-rw-r-- 1 rave rave  1758 14. Mai 09:00 en_GB.gmo
-rw-rw-r-- 1 rave rave  3389 14. Mai 09:00 en_GB.po
-rw-r--r-- 1 rave rave  1203 14. Mai 09:00 en@quot.header
-rw-rw-r-- 1 rave rave  1739 14. Mai 09:00 eo.gmo
-rw-rw-r-- 1 rave rave  3397 14. Mai 09:00 eo.po
-rw-rw-r-- 1 rave rave  1762 14. Mai 09:00 es_AR.gmo
-rw-rw-r-- 1 rave rave  3346 14. Mai 09:00 es_AR.po
-rw-rw-r-- 1 rave rave  1386 14. Mai 09:00 es_CO.gmo
-rw-rw-r-- 1 rave rave  3286 14. Mai 09:00 es_CO.po
-rw-rw-r-- 1 rave rave  1773 14. Mai 09:00 es.gmo
-rw-rw-r-- 1 rave rave  1359 14. Mai 09:00 es_MX.gmo
-rw-rw-r-- 1 rave rave  3237 14. Mai 09:00 es_MX.po
-rw-rw-r-- 1 rave rave  3340 14. Mai 09:00 es.po
-rw-rw-r-- 1 rave rave  1766 14. Mai 09:00 et.gmo
-rw-rw-r-- 1 rave rave  3337 14. Mai 09:00 et.po
-rw-rw-r-- 1 rave rave  1709 14. Mai 09:00 eu.gmo
-rw-rw-r-- 1 rave rave  3406 14. Mai 09:00 eu.po
-rw-rw-r-- 1 rave rave  1436 14. Mai 09:00 fa.gmo
-rw-rw-r-- 1 rave rave  3374 14. Mai 09:00 fa.po
-rw-rw-r-- 1 rave rave  1360 14. Mai 09:00 fi.gmo
-rw-rw-r-- 1 rave rave  3290 14. Mai 09:00 fi.po
-rw-rw-r-- 1 rave rave   554 14. Mai 09:00 fr_CA.gmo
-rw-rw-r-- 1 rave rave  3044 14. Mai 09:00 fr_CA.po
-rw-rw-r-- 1 rave rave  1783 14. Mai 09:00 fr.gmo
-rw-rw-r-- 1 rave rave   954 14. Mai 09:00 frp.gmo
-rw-rw-r-- 1 rave rave  3410 14. Mai 09:00 fr.po
-rw-rw-r-- 1 rave rave  3112 14. Mai 09:00 frp.po
-rw-rw-r-- 1 rave rave   612 14. Mai 09:00 fur.gmo
-rw-rw-r-- 1 rave rave  3073 14. Mai 09:00 fur.po
-rw-rw-r-- 1 rave rave  1190 14. Mai 09:00 fy.gmo
-rw-rw-r-- 1 rave rave  3195 14. Mai 09:00 fy.po
-rw-rw-r-- 1 rave rave  1287 14. Mai 09:00 ga.gmo
-rw-rw-r-- 1 rave rave  3253 14. Mai 09:00 ga.po
-rw-rw-r-- 1 rave rave  1757 14. Mai 09:00 gl.gmo
-rw-rw-r-- 1 rave rave  3393 14. Mai 09:00 gl.po
-rw-rw-r-- 1 rave rave  1567 14. Mai 09:00 gu.gmo
-rw-rw-r-- 1 rave rave  3530 14. Mai 09:00 gu.po
-rw-rw-r-- 1 rave rave  1521 14. Mai 09:00 he.gmo
-rw-rw-r-- 1 rave rave  3400 14. Mai 09:00 he.po
-rw-rw-r-- 1 rave rave  1640 14. Mai 09:00 hi.gmo
-rw-rw-r-- 1 rave rave  3565 14. Mai 09:00 hi.po
-rw-rw-r-- 1 rave rave  1829 14. Mai 09:00 hr.gmo
-rw-rw-r-- 1 rave rave  3407 14. Mai 09:00 hr.po
-rw-rw-r-- 1 rave rave  1776 14. Mai 09:00 hu.gmo
-rw-rw-r-- 1 rave rave  3413 14. Mai 09:00 hu.po
-rw-rw-r-- 1 rave rave  1501 14. Mai 09:00 hy.gmo
-rw-rw-r-- 1 rave rave  3452 14. Mai 09:00 hy.po
-rw-rw-r-- 1 rave rave  1788 14. Mai 09:00 id.gmo
-rw-rw-r-- 1 rave rave  3381 14. Mai 09:00 id.po
-rw-r--r-- 1 rave rave   672 14. Mai 09:00 insert-header.sin
-rw-rw-r-- 1 rave rave  1371 14. Mai 09:00 is.gmo
-rw-rw-r-- 1 rave rave  3243 14. Mai 09:00 is.po
-rw-rw-r-- 1 rave rave  1730 14. Mai 09:00 it.gmo
-rw-rw-r-- 1 rave rave  3341 14. Mai 09:00 it.po
-rw-rw-r-- 1 rave rave  1894 14. Mai 09:00 ja.gmo
-rw-rw-r-- 1 rave rave  3468 14. Mai 09:00 ja.po
-rw-rw-r-- 1 rave rave   666 14. Mai 09:00 ka.gmo
-rw-rw-r-- 1 rave rave  3127 14. Mai 09:00 ka.po
-rw-rw-r-- 1 rave rave  1494 14. Mai 09:00 kk.gmo
-rw-rw-r-- 1 rave rave  3432 14. Mai 09:00 kk.po
-rw-rw-r-- 1 rave rave  1577 14. Mai 09:00 kn.gmo
-rw-rw-r-- 1 rave rave  3540 14. Mai 09:00 kn.po
-rw-rw-r-- 1 rave rave  1858 14. Mai 09:00 ko.gmo
-rw-rw-r-- 1 rave rave  3449 14. Mai 09:00 ko.po
-rw-rw-r-- 1 rave rave   418 14. Mai 09:00 ku.gmo
-rw-rw-r-- 1 rave rave   606 14. Mai 09:00 ku_IQ.gmo
-rw-rw-r-- 1 rave rave  3074 14. Mai 09:00 ku_IQ.po
-rw-rw-r-- 1 rave rave  3757 14. Mai 09:00 ku.po
-rw-rw-r-- 1 rave rave  1413 14. Mai 09:00 ky.gmo
-rw-rw-r-- 1 rave rave  3259 14. Mai 09:00 ky.po
-rw-rw-r-- 1 rave rave   546 14. Mai 09:00 la.gmo
-rw-rw-r-- 1 rave rave  3040 14. Mai 09:00 la.po
-rw-rw-r-- 1 rave rave   415  9. Feb 21:00 LINGUAS
-rw-rw-r-- 1 rave rave  1890 14. Mai 09:00 lt.gmo
-rw-rw-r-- 1 rave rave  3443 14. Mai 09:00 lt.po
-rw-rw-r-- 1 rave rave  1404 14. Mai 09:00 lv.gmo
-rw-rw-r-- 1 rave rave  3339 14. Mai 09:00 lv.po
-rw-rw-r-- 1 rave rave   591 14. Mai 09:00 mai.gmo
-rw-rw-r-- 1 rave rave  3082 14. Mai 09:00 mai.po
-rw-r--r-- 1 rave rave 17577 14. Mai 09:00 Makefile.in.in
-rw-rw-r-- 1 rave rave  3531 10. Mai 11:42 Makevars
-rw-rw-r-- 1 rave rave  2879 14. Mai 09:00 mate-backgrounds.pot
-rw-rw-r-- 1 rave rave  1193 14. Mai 09:00 mg.gmo
-rw-rw-r-- 1 rave rave  3195 14. Mai 09:00 mg.po
-rw-rw-r-- 1 rave rave  1388 14. Mai 09:00 mk.gmo
-rw-rw-r-- 1 rave rave  3392 14. Mai 09:00 mk.po
-rw-rw-r-- 1 rave rave  1717 14. Mai 09:00 ml.gmo
-rw-rw-r-- 1 rave rave  3680 14. Mai 09:00 ml.po
-rw-rw-r-- 1 rave rave  1509 14. Mai 09:00 mn.gmo
-rw-rw-r-- 1 rave rave  3405 14. Mai 09:00 mn.po
-rw-rw-r-- 1 rave rave  1618 14. Mai 09:00 mr.gmo
-rw-rw-r-- 1 rave rave  3548 14. Mai 09:00 mr.po
-rw-rw-r-- 1 rave rave  1768 14. Mai 09:00 ms.gmo
-rw-rw-r-- 1 rave rave  3337 14. Mai 09:00 ms.po
-rw-rw-r-- 1 rave rave  1756 14. Mai 09:00 nb.gmo
-rw-rw-r-- 1 rave rave  3342 14. Mai 09:00 nb.po
-rw-rw-r-- 1 rave rave  1784 14. Mai 09:00 nds.gmo
-rw-rw-r-- 1 rave rave  3420 14. Mai 09:00 nds.po
-rw-rw-r-- 1 rave rave   633 14. Mai 09:00 ne.gmo
-rw-rw-r-- 1 rave rave  3094 14. Mai 09:00 ne.po
-rw-rw-r-- 1 rave rave  1770 14. Mai 09:00 nl.gmo
-rw-rw-r-- 1 rave rave  3402 14. Mai 09:00 nl.po
-rw-rw-r-- 1 rave rave  1239 14. Mai 09:00 nn.gmo
-rw-rw-r-- 1 rave rave  3205 14. Mai 09:00 nn.po
-rw-rw-r-- 1 rave rave  1026 14. Mai 09:00 oc.gmo
-rw-rw-r-- 1 rave rave  3182 14. Mai 09:00 oc.po
-rw-rw-r-- 1 rave rave  1537 14. Mai 09:00 or.gmo
-rw-rw-r-- 1 rave rave  3500 14. Mai 09:00 or.po
-rw-rw-r-- 1 rave rave  1457 14. Mai 09:00 pa.gmo
-rw-rw-r-- 1 rave rave  3423 14. Mai 09:00 pa.po
-rw-rw-r-- 1 rave rave  1879 14. Mai 09:00 pl.gmo
-rw-rw-r-- 1 rave rave  3547 14. Mai 09:00 pl.po
-rw-rw-r-- 1 rave rave    93  1. Feb 2018  POTFILES.in
-rw-rw-r-- 1 rave rave   618 14. Mai 09:00 ps.gmo
-rw-rw-r-- 1 rave rave  3079 14. Mai 09:00 ps.po
-rw-rw-r-- 1 rave rave  1789 14. Mai 09:00 pt_BR.gmo
-rw-rw-r-- 1 rave rave  3422 14. Mai 09:00 pt_BR.po
-rw-rw-r-- 1 rave rave  1772 14. Mai 09:00 pt.gmo
-rw-rw-r-- 1 rave rave  3382 14. Mai 09:00 pt.po
-rw-r--r-- 1 rave rave   153 14. Mai 09:00 quot.sed
-rw-r--r-- 1 rave rave   432 14. Mai 09:00 remove-potcdate.sin
-rw-rw-r-- 1 rave rave  1405 14. Mai 09:00 ro.gmo
-rw-rw-r-- 1 rave rave  3359 14. Mai 09:00 ro.po
-rw-rw-r-- 1 rave rave  2167 14. Mai 09:00 ru.gmo
-rw-r--r-- 1 rave rave  2203 14. Mai 09:00 Rules-quot
-rw-rw-r-- 1 rave rave  3801 14. Mai 09:00 ru.po
-rw-rw-r-- 1 rave rave   479 14. Mai 09:00 rw.gmo
-rw-rw-r-- 1 rave rave  3445 14. Mai 09:00 rw.po
-rw-rw-r-- 1 rave rave   665 14. Mai 09:00 si.gmo
-rw-rw-r-- 1 rave rave  3126 14. Mai 09:00 si.po
-rw-rw-r-- 1 rave rave  1840 14. Mai 09:00 sk.gmo
-rw-rw-r-- 1 rave rave  3422 14. Mai 09:00 sk.po
-rw-rw-r-- 1 rave rave  1380 14. Mai 09:00 sl.gmo
-rw-rw-r-- 1 rave rave  3321 14. Mai 09:00 sl.po
-rw-rw-r-- 1 rave rave  1234 14. Mai 09:00 sq.gmo
-rw-rw-r-- 1 rave rave  3253 14. Mai 09:00 sq.po
-rw-rw-r-- 1 rave rave  1581 14. Mai 09:00 sr.gmo
-rw-rw-r-- 1 rave rave  1338 14. Mai 09:00 sr@latin.gmo
-rw-rw-r-- 1 rave rave  3307 14. Mai 09:00 sr@latin.po
-rw-rw-r-- 1 rave rave  3539 14. Mai 09:00 sr.po
-rw-rw-r-- 1 rave rave    10 14. Mai 09:00 stamp-po
-rw-rw-r-- 1 rave rave  1756 14. Mai 09:00 sv.gmo
-rw-rw-r-- 1 rave rave  3332 14. Mai 09:00 sv.po
-rw-rw-r-- 1 rave rave  1576 14. Mai 09:00 ta.gmo
-rw-rw-r-- 1 rave rave  3539 14. Mai 09:00 ta.po
-rw-rw-r-- 1 rave rave  1632 14. Mai 09:00 te.gmo
-rw-rw-r-- 1 rave rave  3634 14. Mai 09:00 te.po
-rw-rw-r-- 1 rave rave  1437 14. Mai 09:00 th.gmo
-rw-rw-r-- 1 rave rave  3439 14. Mai 09:00 th.po
-rw-rw-r-- 1 rave rave  1762 14. Mai 09:00 tr.gmo
-rw-rw-r-- 1 rave rave  3312 14. Mai 09:00 tr.po
-rw-rw-r-- 1 rave rave   601 14. Mai 09:00 ug.gmo
-rw-rw-r-- 1 rave rave  3083 14. Mai 09:00 ug.po
-rw-rw-r-- 1 rave rave  2226 14. Mai 09:00 uk.gmo
-rw-rw-r-- 1 rave rave  3938 14. Mai 09:00 uk.po
-rw-rw-r-- 1 rave rave   548 14. Mai 09:00 ur.gmo
-rw-rw-r-- 1 rave rave  2993 14. Mai 09:00 ur.po
-rw-rw-r-- 1 rave rave  1218 14. Mai 09:00 uz.gmo
-rw-rw-r-- 1 rave rave  3253 14. Mai 09:00 uz.po
-rw-rw-r-- 1 rave rave  1340 14. Mai 09:00 vi.gmo
-rw-rw-r-- 1 rave rave  3268 14. Mai 09:00 vi.po
-rw-rw-r-- 1 rave rave  1695 14. Mai 09:00 zh_CN.gmo
-rw-rw-r-- 1 rave rave  3273 14. Mai 09:00 zh_CN.po
-rw-rw-r-- 1 rave rave  1219 14. Mai 09:00 zh_HK.gmo
-rw-rw-r-- 1 rave rave  3185 14. Mai 09:00 zh_HK.po
-rw-rw-r-- 1 rave rave  1719 14. Mai 09:00 zh_TW.gmo
-rw-rw-r-- 1 rave rave  3297 14. Mai 09:00 zh_TW.po
``` 
Latest releaesed tarball doesn't ship *.gmo files.

yetist commented:
@raveit65 

Ok, I confirm that the tarball created by `make dist` contains the *.gmo file, and I found the reason from the gettext package.

The gettext package installs a file named `Makefile.in.in`, which is located at `/usr/share/gettext/po/Makefile.in.in` on archlinux.

If the following patch is applied to `Makefile.in.in`, the tarball created by `make dist` does not have a .gmo file.
```
--- old/Makefile.in.in 2019-05-14 15:21:51.455784771 +0800
+++ new/Makefile.in.in     2019-05-14 15:22:09.402334821 +0800
@@ -73,7 +73,7 @@
 DISTFILES.common = Makefile.in.in remove-potcdate.sin \
 $(DISTFILES.common.extra1) $(DISTFILES.common.extra2) $(DISTFILES.common.extra3)
 DISTFILES = $(DISTFILES.common) Makevars POTFILES.in \
-$(POFILES) $(GMOFILES) \
+$(POFILES) \
 $(DISTFILES.extra1) $(DISTFILES.extra2) $(DISTFILES.extra3)
 
 POTFILES = \
```

raveit65 commented:
Issue doesn't exists with using `ninja -C build dist`, so we can ignore this issue.
Seems any other like to test this PR. 
Any way it's a simple package, i will merge it.
@yetist 
Can you adjust travis.yml to use ninja dist after merge, please?
